packages/caspar-graphics/lib/caspar.js

```diff
@@ -25,7 +25,17 @@ export default class Caspar extends React.Component {
     this.state = {
       state: States.loading,
       data: props.data || getQueryData(),
-      didError: false
+      didError: false,
+      visibleReference: false,
+      referenceImage: null
+    }
+
+    try {
```

Comment:
Could we do this only in dev mode?

Reply:
the whole toggle thing?

Reply:
yes, don't want to run the risk of showing it in production.